cmd/nlb_list.go

```diff
@@ -70,19 +71,22 @@ func (c *nlbListCmd) cmdRun(_ *cobra.Command, _ []string) error {
 		done <- struct{}{}
 	}()
 	err := utils.ForEachZone(zones, func(zone string) error {
-		ctx := exoapi.WithEndpoint(gContext, exoapi.NewReqEndpoint(account.CurrentAccount.Environment, zone))
+		client, err := switchClientZoneV3(ctx, globalstate.EgoscaleV3Client, v3.ZoneName(zone))
+		if err != nil {
+			return err
+		}
 
-		list, err := globalstate.EgoscaleClient.ListNetworkLoadBalancers(ctx, zone)
+		list, err := client.ListLoadBalancers(ctx)
 		if err != nil {
 			return fmt.Errorf("unable to list Network Load Balancers in zone %s: %w", zone, err)
 		}
 
-		for _, nlb := range list {
+		for _, nlb := range list.LoadBalancers {
 			res <- nlbListItemOutput{
-				ID:        *nlb.ID,
-				Name:      *nlb.Name,
+				ID:        nlb.ID.String(),
```

Comment:
like here for `ID`

Reply:
While I conceptually agree and it's tempting to have the types of the output fields match the data struct fields, we end up typecasting those values as string anyway while building the table; i'll keep it for `Zone` for now as it's used when building the client